controllers/components/kueue/kueue_support.go

```diff
@@ -33,3 +37,17 @@ func manifestsPath() odhtypes.ManifestInfo {
 		SourcePath: "rhoai",
 	}
 }
+
+func extramanifestsPath() odhtypes.ManifestInfo {
+	return odhtypes.ManifestInfo{
+		Path:       odhdeploy.DefaultManifestPath,
+		ContextDir: ComponentName,
+		SourcePath: "rhoai/ocp-4.17-addons",
+	}
+}
+
+// return true if OCP is greater or equal 4.17.
+func vapPredicate(context.Context, *odhtypes.ReconciliationRequest) bool {
+	var V417 = semver.MustParse("4.17")
```

Comment:
I'd move the semver calculation out of the method, so It does not need to be recomputed for every predicate invocation

Reply:
updated